example-config.yaml

```diff
@@ -1,7 +1,15 @@
+# You will need to use a custom emoji here for when the bot is processing the image.
+# Get rid of the "a" if you are not using an animated emoji
 queue_emoji: a:loading:572202235342225418
 
+# This will be the guild where the bot will be serving blurple images.
 project_blurple_guild: 559341262302347314
 
+# Roles for when you have rolled for a team (either dark or light)
+# To set this up:
+# 1. Create 5 roles
+# 2. Make them mentionable temporarily
+# 3. Grab their ID's (put a \ before the @ when mentioning the role.)
```

Comment:
Role IDs can now be obtained by right clicking on roles in the role menu or in a users profile.

Reply:
I didn't know that until now, I'll add that as well.